### PR TITLE
mlflow implemetation in biencoder

### PR DESCRIPTION
**Pull request by SatyamRajawat**
modified:   jaseci_kit/jaseci_kit/modules/encoders/bi.py
	modified:   jaseci_kit/jaseci_kit/modules/encoders/utils/model_config.json
	modified:   jaseci_kit/jaseci_kit/modules/encoders/utils/train.py
	modified:   jaseci_kit/jaseci_kit/modules/encoders/utils/train_config.json

### CODE REVIEW

File: jaseci_kit/jaseci_kit/modules/encoders/bi_enc.py [hunk 1]
```diff
@@ -190,74 +183,107 @@ def infer(
...
+        train_config=train_config,
+    )
+    return model
+
+
 # API for training
 @jaseci_action(act_group=["bi_enc"], allow_remote=True)
-def train(dataset: Dict = None, from_scratch=False, training_parameters: Dict = None):
+def train(
+    use_mlflow: bool = True,
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/encoders/bi_enc.py:212]: We have to make use_mlflow =false

**Review by AshishMahendra — COMMENTED** [review]
Rest all looks fine, old testing also works

**Review by AshishMahendra — APPROVED** [review]

File: jaseci_kit/jaseci_kit/modules/encoders/bi_enc.py (hunk 1 above)
Comment (ypkang) [reply jaseci_kit/jaseci_kit/modules/encoders/bi_enc.py:212]: okay are you making this change or are you expecting me to?

Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/modules/encoders/bi_enc.py:212]: this is from satyam's repository, not sure how to update this

**Comment by ypkang** [comment]
@AshishMahendra can you copy the code from Sataym's repo to yours and make a new PR so that we can update it?

**Comment by AshishMahendra** [comment]
> @AshishMahendra can you copy the code from Sataym's repo to yours and make a new PR so that we can update it?

Sure, i'll do that

**Comment by ypkang** [comment]
Replaced by #357

**Pull request closed without merging**